### ISSUE TITLE
can not import name 'xr' from 'ismn.const' in ismn-1.5.0

### ISSUE THREAD

**Issue by Dushuai12138** [TUW-GEO/ismn#89]
I installed ismn-1.5.0, ran ```from ismn.interface import ISMN_Interface``` and got bug
```
......
36 from ismn.const import xarray_available, xr      
38 import json     
41 class IsmnComponent:  
ImportError: cannot import name 'xr' from 'ismn.const'(E:\ProgramData\Anaconda3\envs\gee1\Lib\site-packages\ismn\const.py)
```
I uninstalled ismn-1.5.0 and installed ismn-1.4.0, this error disapeared.

**Comment by wpreimes** [comment]
Thanks for reporting and sorry for the issue. xarray should be an optional requirement, but right now it is required it seems. I will fix this immediately, in the meantime, as a workaround, you can install xarray (e.g conda install xarray), see also https://docs.xarray.dev/en/stable/getting-started-guide/installing.html

**Comment by wpreimes** [comment]
I have just released v1.5.1 which should fix this issue.

**Comment by Dushuai12138** [comment]
> I have just released v1.5.1 which should fix this issue.

yeah, it works. Thanks.
